client/src/components/ReactAdminBoostrap.js

```diff
@@ -0,0 +1,13 @@
+/* global reactAdminConfig */
+import React from 'react';
+import { loadComponent } from 'lib/Injector';
+
+const ReactAdminBoostrap = () => {
+  const RootComponent = loadComponent(reactAdminConfig.component);
+  const { props } = reactAdminConfig;
+  return (
+    <RootComponent {...props} />
```

Comment:
We load the components and props as defined by the ReactAdmin PHP implementation.